redesign/generator_builder.py

```diff
@@ -0,0 +1,30 @@
+"""Contains generator factory class that picks generator based on its profile type."""
+
+from generators import GraphGenerator, TabularGenerator, UnstructuredGenerator
+from dataprofiler import StructuredProfiler, UnstructuredProfiler #, GraphProfiler
+
+class Generator:
+    """Generator class."""
+
+    valid_data_types = {
+        StructuredProfiler: TabularGenerator,
+        #GraphProfiler: GraphGenerator, 
```

Comment:
Iif it isn't ready yet, let's just leave it out as opposed to commented. We can add it when it is setup.

Reply:
same as above in the imports